sealtk/noaa/core/ImageListVideoSourceFactory.cpp

```diff
@@ -52,9 +54,13 @@ kwiver::vital::config_block_sptr ImageListVideoSourceFactory::config(
   auto config = kwiver::vital::config_block::empty_config();
   config->set_value("video_reader:type", "image_list");
   config->set_value("video_reader:image_list:image_reader:type",
-                    "noaa_timestamp_passthrough");
-  config->set_value("video_reader:image_list:image_reader:"
-                    "noaa_timestamp_passthrough:image_reader:type", "vxl");
+                    config::videoReader);
+  if (*config::videoReaderPassthrough)
```

Comment:
It's not clear to me why we check for this value but assume the other 3 will exist?

Reply:
If the others aren't set, you're producing an unusable binary. Using a pass-through video reader however is not required; conceivably we could be using a video reader that does not use a secondary reader type (e.g. GDAL), in which case setting the passthrough reader type is at best superfluous. (I *think* unknown config keys are ignored, but...)

Reply:
Put differently, this one is optional, while the others, practically speaking, aren't.

Reply:
Ok